_posts/2018-10-29-bazel-0.19.0.md

```diff
@@ -0,0 +1,91 @@
+
+* The `--loading_phase_threads` flag now defaults to `auto` instead of 200, which at the moment corresponds to the number of CPUs. This is appropriate for most users. However, if your sources reside on a network file system, increasing this value may yield better analysis-time performance when disk caches are cold.
+
+## Android
+
+* Fixed missing debug symbols when building native code with `--compilation_mode=dbg` that target Android ARM architectures by adding the `-g` flag.
+
+## C++
+
+* Added `--incompatible_disable_legacy_flags_cc_toolchain_api` to deprecate legacy `cc_toolchain` Starlark API for legacy `CROSSTOOL` fields. Tracking issue is #6434. See the instructions for migration.
+
+* Runfiles in `cc_test`: the C++ runfiles library (`@bazel_tools//tools/cpp/runfiles`) can now create Runfiles objects for tests. See `//tools/cpp/runfiles/runfiles_src.h` in the Bazel source tree for documentation.
+
+* :cc_binary link action no longer hardcodes `-static-libgcc` for toolchains that support embedded runtimes (guarded by `--experimental_dont_emit_static_libgcc` temporarily).
+
+* The flag `--experimental_enable_cc_configuration_make_variables` is removed, use `--incompatible_disable_cc_configuration_make_variables` instead.
+
+## Java
+
+* Bazel now requires a JDK to be installed as it no longer falls back to the internal JDK if the `--javabase` flag is not set. Instead, Bazel now searches for a JDK within the `JAVA_HOME` and `PATH` environment variables. If Bazel cannot find a JDK, it will not be able to run Java builds. 
+
+## Code Coverage
+
+* LcovMerger was renamed to [CoverageOutputGenerator](https://github.com/bazelbuild/bazel/pull/6137). Please use 
+```
+--coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main
+```
+
+instead of the previous 
+
+```
+--coverage_report_generator=@bazel_tools//tools/test/LcovMerger/java/com/google/devtools/lcovmerger:Main
+```
+
+
+* You can now enable a new coverage method for C++ rules that use gcc with the --experimental_cc_coverage flag. The flag will use gcov instead of lcov to collect coverage information. This new implementation fixes
+some of the C++ coverage correctness issues ( [respecting `--instrumentation_filter` and `--instrument_test_targets` flags](https://github.com/bazelbuild/bazel/issues/6129) ). 
+There have been observed faster clean and incremental times when invoking bazel coverage ( [small overhead compared to test runtimes instead of 2x test runtime](https://github.com/bazelbuild/bazel/issues/5882) ),  but more measurements need to be done.
+
+
+## Other Changes
+
+* Add `--apple_compiler` and `--apple_grte_top options`. These provide the equivalent of `--compiler` / `--grte_top` for the toolchain configured in `--apple_crosstool_top`.
+
+* There is now a `same_pkg_direct_rdeps` query function. See the query documentation for more details.
```

Comment:
link to query documentation?

Reply:
Does this need to be fixed before we can announce the release?

Reply:
No, I don't think so.